lesson06/assignment/pranayama.html

```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+  <link rel="stylesheet" href="css/style.css" />
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css">
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.16.0/umd/popper.min.js"></script>
+  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.5.2/js/bootstrap.min.js"></script>
+
+  <title>Assignment 06 - Bootstrap 4</title>
+</head>
+<body>
+  <div class="grid-container">
+    <div class="header">
+      <header>
+        <div class="logo">
+          <img src="image/yoga-chakra.jpg" id ="yoga-chakra" alt="man sitting in ayogic posture">
+        </div>
+        <h1>Yoga</h1>
+      </header>
+    </div>
+    <div class="nav-bar">
+      <nav>
+        <ul class="nav-links">
+          <li class="link"><a href="index.html">Home</a></li>
+          <li class="link"><a href="about.html">About Us</a></li>
+          <li class="link"><a href="yoga.html">Yoga</a></li>
+          <li class="link"><a href="pranayama.html">Pranayama</a></li>
+
+
+        </ul>
+      </nav>
+    </div>
+    <div>
+      <main class="main">
+
+        <div class="container">
+          <h2>Pranayama: The Yogic Breathing</h2>
+          <p>If you’re feeling tired, heavy, dull, or emotionally off-balance, a few minutes of yogic breathing can energize, renew, and sustain you through your day or a challenging yoga class. There are a number of different breathing techniques in yoga that can influence and affect your experience in asana, relaxation, and meditation. Having a stronger connection and control of your breath will give you a deeper attunement of your physical, mental, and emotional bodies and help anchor your awareness in the present moment. Yogis believe that if you can master your breath through pranayama practices you can master your mind!</p>
+          <div class="row">
+            <div class="col-md-4">
```

Comment:
Per the instruction requirements, we'd like you to implement at least three bootstrap sizes. "md" counts as one, but we'd like to see two more.